mantatail.py

```
@@ -48,15 +60,22 @@ def delete_user(self, nick: str) -> None:
         Note: This does not actually disconnect the user from the server.
         To disconnect the user, a tuple (None, None) must be put in their send queue.
         """
-        user = self.connected_users[nick.lower()]
-        for channel in self.channels.values():
-            if user in channel.users:
-                channel.users.discard(user)
-        del self.connected_users[nick.lower()]
+        user = self.find_user(nick)
+        if user:
+            for channel in self.channels.values():
+                if user in channel.users:
+                    channel.users.discard(user)
+            del self.connected_users[nick.lower()]
 
     def delete_channel(self, channel_name: str) -> None:
-        """Removes a channel from server."""
-        del self.channels[channel_name.lower()]
+        """
+        Removes a channel from server.
+        Silently ignores a non-existing channel.
```

Why is silently ignoring a good thing?

---

It's not necessarily a good or bad thing. I didn't know how else to treat it.

The user never deliberately deletes a channel. If they `PART` as the last user on the channel, it is deleted.
Therefore, there should never be a `KeyError`.. which I realize now, is a good reason to instead `raise` an error if it would happen anyways.